tests/hpc/ww4.pm

```diff
@@ -45,9 +45,9 @@ sub run ($self) {
     record_info "warewulf.conf", script_output("cat /etc/warewulf/warewulf.conf");
     record_info "defaults.conf", script_output("cat /etc/warewulf/defaults.conf");
 
-    $rt = (assert_script_run "wwctl container import docker://registry.suse.de/suse/containers/sle-micro/5.3/containers/suse/sle-micro-rancher/5.3:latest sle-micro-5.3 --setdefault", timeout => 320) ? 1 : 0;
+    $rt = (assert_script_run "wwctl container import docker://registry.suse.de/home/eeich/sle-hpc-15-sp5-container/containers/suse/sle-hpc-node:latest sle-hpc --setdefault", timeout => 320) ? 1 : 0;
```

Comment:
oh gosh are we really gonna rely on home/eeich ?

I don't like such approach. Maybe to mitigate the bus factor, we could at least have a test variable for this?

Reply:
wouldn't be some openSUSE based container better  or even smaller one like alpine linux ?

Reply:
@schlad, a final solution is still WIP. I can try to create this in Devel:HPC - but the idea to have a variable is valid as the container is SP5 dependent. 
@czerw no. This is a *special* container meant for ww4 - it contains a kernel, systemd, identifies as SLE HPC and is built as a base container for compute node deployment. 
An alpine linux is *not* a special container for ww4.